examples/asr/experimental/wav2vec/configs/wav2vec_pretrain.yaml

```diff
@@ -0,0 +1,122 @@
+name: &name "Wav2vec_Pretrain"
+
+model:
+  wav2vec:
+    loss:
+      prob_ppl_weight: 0.1
+      feature_loss_weight: 0
+    quantize:
+      quantize_targets: True
+      quantize_input: False
+      same_quantizer: False
+      latent_vars: 320
+      latent_groups: 2
+      latent_dim: 0
+      latent_temp: [2, 0.5, 0.999995]
+    conv_feature_encoder:
+      extractor_mode: default
+      conv_bias: False
+    transformer_encoder:
+      dropout: 0.1
+      conv:
```

Comment:
Is this conv different from conv_feature_encoder?

Reply:
yeah, `conv` is used to generate the positional embeddings for the transformer layers (using a Conv model). The Conv Feature Encoder is used to extract features from raw audio. I added this explanation to the conf :)